[202405][DNX] - `route/test_route_perf.py` fails with `l3_alpm_enable` missing from `config.bcm` files on DNX systems

### Issue Description

`route/test_route_perf.py` checks the soc property of the system via `conf show l3_alpm_enable`.
On DNX systems we don't include `l3_alpm_enable` in the soc properties (config.bcm) files therefore the test fails.

We confirmed with BRCM in CS00012377343 that `l3_alpm_enable` is specific to XGS systems

### Results you see

route/test_route_perf.py:100: AttributeError</error></testcase><testcase classname="route.test_route_perf" name="test_perf_add_remove_routes[6-cmp206-6-None]" file="route/test_route_perf.py" line="214" time="16.666"><properties><property name="start" value="2024-11-06 11:36:07.320016" /><property name="end" value="2024-11-06 11:36:23.996828" /></                properties><error message="failed on setup with &quot;Failed: l3_alpm_enable is not set for route scaling&quot;">duthosts = [&lt;MultiAsicSonicHost cmp206-4&gt;, &lt;MultiAsicSonicHost cmp206-5&gt;, &lt;MultiAsicSonicHost cmp206-6&gt;, &lt;MultiAsicSonicHost cmp206&gt;], enum_rand_one_per_hwsku_frontend_hostname = 'cmp206-6', tbinfo = {'auto_recover': True,     'comment': 'Tests Arista Arista-7804R3-FM', 'conf-name': 'ardut', 'duts': ['cmp206-4', 'cmp206-5', 'cmp206-6', 'cmp206'], ...}                
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              
    @pytest.fixture
    def check_config(duthosts, enum_rand_one_per_hwsku_frontend_hostname, tbinfo):
        if tbinfo["topo"]["type"] in ["m0", "mx"]:
            return

        duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
        asic = duthost.facts["asic_type"]
        asic_name = duthost.get_asic_name()
        if (asic == "broadcom" and asic_name not in ['th4', 'th5']):
            alpm_enable = duthost.command('bcmcmd "conf show l3_alpm_enable"')["stdout_lines"][2].strip()
            logger.info("Checking config: {}".format(alpm_enable))
&gt;           pytest_assert(alpm_enable == "l3_alpm_enable=2", "l3_alpm_enable is not set for route scaling")
E           Failed: l3_alpm_enable is not set for route scaling

alpm_enable = 'drivshell&gt;'
asic       = 'broadcom'
asic_name  = 'unknown'
duthost    = &lt;MultiAsicSonicHost cmp206-6&gt;
duthosts   = [&lt;MultiAsicSonicHost cmp206-4&gt;, &lt;MultiAsicSonicHost cmp206-5&gt;, &lt;MultiAsicSonicHost cmp206-6&gt;, &lt;MultiAsicSonicHost cmp206&gt;]
enum_rand_one_per_hwsku_frontend_hostname = 'cmp206-6'
tbinfo     = {'auto_recover': True, 'comment': 'Tests Arista Arista-7804R3-FM', 'conf-name': 'ardut', 'duts': ['cmp206-4', 'cmp206-5', 'cmp206-6', 'cmp206'], ...}

### Results you expected to see

Test should've skipped the soc property check

### Is it platform specific

broadcom

### Relevant log output

_No response_

### Output of `show version`

_No response_

### Attach files (if any)

_No response_

--- a/tests/route/test_route_perf.py
+++ b/tests/route/test_route_perf.py
@@ -60,6 +60,10 @@ def check_config(duthosts, enum_rand_one_per_hwsku_frontend_hostname, enum_rand_
         return
 
     duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+    if (duthost.facts.get('platform_asic') == 'broadcom-dnx'):
+        # CS00012377343 - l3_alpm_enable isn't supported on dnx
+        return
+
     asic = duthost.facts["asic_type"]
     asic_id = enum_rand_one_frontend_asic_index
 

